EcoSoapBank/Network Controller/GraphQLController.swift

```diff
@@ -32,6 +36,10 @@ class GraphQLController {
         self.session = session
     }
 
+    func provideToken(_ token: String) {
+        self.token = token
+    }
+
```

Comment:
This all looks good. I'll add some logic in the request method to add the token in the header if it exists once I confirm for sure that's how it gets added.